lang/it.json

```diff
@@ -8,7 +8,7 @@
 	"DUMP": "Estrai %ls",
 	"DUMPING": "Sto estraendo %ls to\n%s...\n",
```

Comment:
That's because I translated in both languages :stuck_out_tongue_winking_eye: 
It's actually "en" in Spanish and "su" in Italian.
Anyway, I've already fixed that in the latest commit :wink: